docs/ManuallyRunning.md

```diff
@@ -0,0 +1,264 @@
+```
+
+Or:
+
+```shell script
+$ init-mongo --mongo-uri localhost:27017 create-indexes
+```
+
+If you are using an Atlas instance which includes authentication, then you would use something like the following:
+
+```shell script
+export SELECTED_TESTS_MONGO_URI=mongodb+srv://<username>:<password>@<atlas>.mongodb.net/
+$ init-mongo create-indexes
+``` 
+
+Refer to the [Atlas connect to cluster](https://docs.atlas.mongodb.com/connect-to-cluster/) for instructions on how to access your cluster uri.  
+
+**Note** from here on, we will assume that **SELECTED_TESTS_MONGO_URI** is set.
+
+## Launch Web Service
+
+```shell script
+$ export EVG_API_USER=$(python -c "import yaml; import sys; cfg = yaml.safe_load(sys.stdin); print(cfg['user'])" < ~/.evergreen.yml)
+$ export EVG_API_KEY=$(python -c "import yaml; import sys; cfg = yaml.safe_load(sys.stdin); print(cfg['api_key'])" < ~/.evergreen.yml)
+$ export SELECTED_TESTS_MONGO_URI="localhost:27017"
+$ uvicorn --host 0.0.0.0 --port 8080 --workers 1 selectedtests.app.asgi:app --reload
+```
+
+__Note__: reload is only used in development mode.
+
+## <a name="GenerateTestTaskMappings"></a>Generate test and task mappings 
+
+Use the following commands to create the test and task mappings for **mongodb-mongo-master**.
+
+```shell script
+$ export SELECTED_TESTS_URI=http://localhost:8080
+$ export SELECTED_TESTS_PROJECT=mongodb-mongo-master 
+$ curl  -X POST ${SELECTED_TESTS_URI}/projects/${SELECTED_TESTS_PROJECT}/test-mappings \
+        -d '{"source_file_regex":"^src/mongo", "test_file_regex":"^jstests"}' \
+        -H "Content-Type: application/json"
+$ curl -X POST ${SELECTED_TESTS_URI}/projects/${SELECTED_TESTS_PROJECT}/task-mappings \
+       -H 'accept: application/json' \
+       -H 'Content-Type: application/json' \
+       -d '{ "source_file_regex": "^src/mongo", "module": "enterprise", "module_source_file_regex": "^src", "build_variant_regex": "^!"}'
+``` 
+
+Now that the mappings have been created, we must populate them.
+
+## Populating the mappings.
+ 
+You should run the `process-test-mappings` and `process-task-mappings` commands once every
+day (see [Generate test and task mappings](#GenerateTestTaskMappings)) . This will gather the
+unprocessed test mapping create and task mapping create requests and process them so that test and
+task mappings for those projects are added to the db.
+
+You should run the `test-mappings update` and `task-mappings update` daily to update the test and
+task mappings models. These jobs look at all git commits and mainline patch builds from the previous
+day and create new test mappings and task mappings respectively.
+
+Assuming your [github keys](#GithubAccess) were created and are in  **~/.ssh/selected_tests_rsa**,
+you can run the following commands.
+
+```shell script
+$ eval "$(ssh-agent -s)"
```

Comment:
The ticket refers to GITHUB_PRIVATE_KEY / GITHUB_PUBLIC_KEY and the cronjob script. But I don't think we can use the scripts here (as they fail if the ssh key fille exist).

The following avoids this issue, but it does put the onus on the developer.